edl_controller.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: training-job-controller
+  namespace: paddlecloud
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: training-job-controller
+    spec:
+      containers:
+      - name: training-job-controller
+        image: paddlepaddle/edl-controller
+        imagePullPolicy: Never
```

Comment:
should we change the policy to `IfNotPresent`?

Reply:
Done.